bedrock/configuration-files.md

```diff
@@ -11,13 +11,17 @@ published: true
 ---
 The root `web/wp-config.php` is required by WordPress and is only used to load the other main configs. Nothing else should be added to it.
 
-`config/application.php` is the main config file that contains what `wp-config.php` usually would. Base options should be set in there.
+The file to modify for configuration options is `config/application.php`. This file that contains what `wp-config.php` usually would.
```

Comment:
typo: "This is the file that contains..." i assume? Or "This file contains ..."

Reply:
Yes. "This is the file that contains..."